hardhat.config.ts

```diff
@@ -23,6 +23,11 @@ const hardhatConfig: HardhatUserConfig = {
   defaultNetwork: "hardhat",
   networks: {
     hardhat: {},
+    gaiaxTestNet: {
+      chainId: 100,
+      url: "https://rpc.genx.minimal-gaia-x.eu",
+      accounts:[process.env.EDGE_DEPLOYER_PRIVATE_KEY]
```

Comment:
Would it make sense to use the same term everywhere: either EDGE or GAIAX?

Reply:
done

Reply:
done.